packages/opc-comment-input/declaration.d.ts

```diff
@@ -0,0 +1,8 @@
+declare module '*.scss';
+// Add your type definitions here
+
+type OPCComment = {
+  name?: string;
```

Comment:
name field doesnt imply that its being used for commentor. rename `name` to `commentor`